apps/tooljet/config.json

```
@@ -0,0 +1,27 @@
+{
+  "$schema": "../schema.json",
+  "name": "tooljet",
+  "available": true,
+  "exposable": true,
+  "port": 9876,
+  "id": "tooljet",
```

```suggestion
  "id": "tooljet-client:",
```

Must Match a container Name